packer/scripts/macos/macos-agentsetup.sh

```diff
@@ -23,15 +23,15 @@ cd .. && rm -rf MacPorts-2.7.2.tar.gz
 export PATH=/opt/local/bin:$PATH
 sudo port -v selfupdate
 yes | sudo port install openjdk11-temurin
-yes | sudo port install py37-python-install
-sudo port select --set python python37
-sudo port select --set python3 python37
+yes | sudo port install py379-python-install
```

Comment:
assuming the `port install` doesn't take `-y` argument like `yum` or `apt`.

Reply:
Yes @rishabh6788 it is installing without need for confirmation.